R/tmb-methods.R

```diff
@@ -625,7 +625,7 @@ simulate.mmrm_tmb <- function(object,
         # Recalculate betas with sampled thetas.
         hold <- object$tmb_object$report(newtheta)
         # Resample betas given new beta distribution.
-        beta_sample <- hold$beta + VCOV %*% rnorm(length(hold$beta))
+        beta_sample <- hold$beta + t(chol(hold$beta_vcov)) %*% rnorm(length(hold$beta))
```

Comment:
For clarity, this is the more time-consuming approach, as I understand it, in which we directly use the covariance matrix `beta_vcov` as provided by `report()`.

Reply:
this is time consuming but it seems that we have to include this to sample $\beta$